frontend/src/components/uam/common/types/users-table-actions-props.type.ts

```diff
@@ -1,6 +1,7 @@
 type UsersTableActionsProps = {
   id: number;
   onDelete: (id: number) => void;
+  hasSameUser: boolean;
```

Comment:
this name confused me, maybe we should mention it's actually related to "current logged in user"